proposals/testing/XXXX-issue-severity-warning.md

```diff
@@ -0,0 +1,113 @@
+# Test Issue Severity
+
+* Proposal: [ST-XXXX](XXXX-issue-severity-warning.md)
+* Authors: [Suzy Ratcliff](https://github.com/suzannaratcliff)
+* Review Manager: TBD
+* Status: **Pitched**
+* Bug: [swiftlang/swift-testing#1075](https://github.com/swiftlang/swift-testing/pull/1075)
+* Implementation: [swiftlang/swift-testing#1075](https://github.com/swiftlang/swift-testing/pull/1075)
+
```

Comment:
Can you fill out the "Review" header field starting with a link to the Pitch thread on the Forums

```markdown
* Review: ([pitch](https://forums.swift.org/...))
```